manifests/snmp_listen_ip.pp

```diff
@@ -0,0 +1,13 @@
+class squid::snmp_listen_ip (
```

Comment:
Having changed it to a hash, this now needs to be a `defined type` instead of a class.